subcommand/server-acl-init/command.go

```diff
@@ -491,6 +501,108 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	if len(c.flagIngressGatewayNames) > 0 {
```

Comment:
What do you think about reducing the duplication here by pulling this code into a generic function for both gateway types:
```go
code := c.createGatewayPolices(
  "ingress-gateway",
  c.flagIngressGatewayNames,
  c.ingressGatewayRules,
  consulDC,
  consulClient)

if code != 0 {
  return code
}

code = c.createGatewayPolices(
  "terminating-gateway",
  c.flagTerminatingGatewayNames,
  c.terminatingGatewayRules,
  consulDC,
  consulClient)

if code != 0 {
  return code
}

// generic func signature
func (c *Command) createGatewayPolices(
	gatewayType string,
	names []string,
	rulesF func(string, string) (string, error),
	consulDC string,
	consulClient *api.Client) int {...}
```